pkg/driver/aws-efs/aws_efs.go

```diff
@@ -243,3 +248,50 @@ func withVolumeMetricsDaemonSetHook(c *clients.Clients) (csidrivernodeservicecon
 	return hook, informers
 
 }
+
+// withCustomTags add tags from Infrastructure.Status.PlatformStatus.AWS.ResourceTags to the driver command line as
+// --tags=<key1>:<value1> <key2>:<value2>,...
+func withCustomTags(c *clients.Clients) (dc.DeploymentHookFunc, []factory.Informer) {
+	hook := func(spec *opv1.OperatorSpec, deployment *appsv1.Deployment) error {
+		infraLister := c.GetInfraInformer().Lister()
+		infra, err := infraLister.Get(infrastructureName)
+		if err != nil {
```

Comment:
I would move this code to a different function and perform this computation only if container in question is `csi-driver`